README.md

```diff
@@ -16,3 +16,19 @@ To apply the `kustomization` command locally, to verify what is the output of th
 cd manifests/registry
 oc get -k ./overlay_limitador/ -o yaml | tee log.txt
 ```
+
+## Multitenancy enabled deployment
+
+The `overlay_multitenancy` contains configuration files for deploying Apicurio Registry with multitenancy enabled.
+
+```bash
+./kustomize build ./manifests/registry/overlay_multitenancy/ | k apply -f -
```

Comment:
what's `k`?

Reply:
the command could be changed to `oc apply -k ./manifests/registry/overlay_multitenancy/`

Reply:
my bad, `k` is my alias for `kubectl`